CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Unreleased
+
+* [Enhancement] If `S3_PROFILE_IMAGE_BUCKET` is not set, store profile 
+  images to the filesystem. This is the new default behavior. If a bucket 
+  name is provided, upload profile images to S3.
+* [Fix] Add a dummy `base_url` to profile image backend to prevent
+  it from crashing LMS when running with `DEBUG=True`.
```

Comment:
I'd add something like "(such as when using `tutor dev`)" at the end, because I don't think it's obvious to everyone that `tutor dev` enables `DEBUG`. Does that make sense?

Reply:
Makes sense. I added it.